### PR TITLE
Roll chrome 96 attempt 2

### PR DESCRIPTION
**Pull request by yjbanov**
Roll Chromium 96.2.

This is the second attempt to roll it. The previous attempt (https://github.com/flutter/flutter/pull/99725) failed due to a LUCI cache issue.

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ platform_properties:
         ]
       os: Linux
       device_type: none
+      clobber: "true"
```
Comment (CaseyHillers) [inline .ci.yaml:31]: Unfortunately, chrome_and_driver doesn't take in clobber: true. https://cs.opensource.google/flutter/recipes/+/main:recipe_modules/flutter_deps/api.py;l=146

I think we'll want to wait for https://github.com/flutter/flutter/pull/100072 to remove the swarming cache altogether

**Review by godofredoc — APPROVED** [review]

File: .ci.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ platform_properties:
         [
           {"name":"builder_linux_framework","path":"builder"},
           {"name":"android_sdk","path":"android"},
-          {"name":"chrome_and_driver","path":"chrome"},
+          {"name":"chrome_and_driver","path":"chrome_96"},
```
Comment (godofredoc) [inline .ci.yaml:20]: Actually we only need to change the name:

`{"name":"chrome_and_driver_96","path":"chrome"},`

The path is fixed and mounted at runtime from the named cache.

Comment (yjbanov) [reply .ci.yaml:20]: Changing the name results in recipe failures, so I can't use that method.

Comment (godofredoc) [reply .ci.yaml:20]: Can you please share the one with recipe failures to take a look?

Comment (godofredoc) [reply .ci.yaml]: We are still missing to provide the version on all the chrome_and_driver dependencies.

**Comment by skia-gold** [comment]
Gold has detected about 25 new digest(s) on patchset 3.
View them at https://flutter-gold.skia.org/cl/github/100073

**Comment by skia-gold** [comment]
Gold has detected about 14 new digest(s) on patchset 4.
View them at https://flutter-gold.skia.org/cl/github/100073

**Pull request merged**